### ISSUE TITLE
option to as default open novel in external epub reader

### ISSUE THREAD

**Issue by rpcarvalheira** [LNReader/lnreader#718]
### Describe your suggested feature

TL;DR - add the option to directly open the downloaded epub directly in an external epub reader.

Hello! It's been a long time. I utilize LNReader as well as Shousetsu and Quicknovel. To be fair, each and every one of those has its own advantages and disavantages. 

Shousetsu has extensions but low development and a small community, while it works, lots of features require more knowledge (like the fact that you cannot fully customise without having to do it in CSS - and its kinda fast, but without exporting to epub.
QuickNovel is a good tool, it works good most of the times (sometimes the downloads gets buggy, but updates the exported epubs directly and offers to as default open them in your (selected) external epub reader. 

And for me, LNReader has a bigger community, it doesn't get so buggy in downloads, updating epubs is less buggy than QuickNovel; however, there is this lack of the possibility to directly opening the books in external readers and while I don't have much of a problem with that, the lack of this feature makes me lose some time opening it in my downloads and then opening the file in another reader.

Why? Well, while the reading experience itself is getting better in the app, not just me but almost everyone in the LN universe that reads in a phone or tablet have apps dedicated to reading, with easy access of customisation, like Moon Reader+. Their accessibility settings makes it easier for me (as neurodivergent and visually impaired) to improve the readability while customising the whole experience to be tailored to my necessities. So, while using the app to both download from lots of places is really good, most of the times I prefer to use QuickNovel to make it easier, specially at night, since it directly opens it in moonreader (selected as default to open the epub in the external reader).

And to be fair, while I know this is rather clumsy or egoistic to ask that jut for me, I do think that this is so much of a good feature to implement that makes me use as default another app while yours are better in lots of aspects but this - which makes me rather sad - but can as well impact positively other people with disabilities. Other than the reading capabilities of said app (MoonReader), it's interface sucks where LNReader is way better.

### Other details

I know that this will impact and probably disable integrations like MAL and AniList if selected, so could there be a warning of that as well :).

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/LNReader/lnreader-sources/issues/new/choose).
- [X] I have updated the app to version **[1.1.14](https://github.com/LNReader/lnreader/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by ADARSHPR0** [comment]
Huh totally agree sad to see no such features is here yet
